community/join.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 3
+---
+
+# Join the team
+
+conda-forge is a volunteer-driven community. We would love for you to
+help us.
+
+We have three main groups of people who maintain the ecosystem. They are
+the **Core Team**, the **staged-recipes Maintainers**, and the
+**Feedstock Maintainers**. If you are interested in helping us maintain
+this ecosystem, our biggest need is help in reviewing new conda packages
+coming in through staged-recipes.
+
+## The Core Team
```

Comment:
```suggestion
## The Core Team {#core}
```

This syntax allows us to redefine the anchor tag.

Reply:
Ohhh nice :)